crates/vm/src/system/memory/volatile/mod.rs

```diff
@@ -269,7 +258,11 @@ impl<F: PrimeField32> ChipUsageGetter for VolatileBoundaryChip<F> {
     }
 
     fn current_trace_height(&self) -> usize {
```

Comment:
actually VolatileBoundary = no continuations, so `current_trace_height` is never going to matter because we don't segment

Reply:
good point